examples/04_glm_first_level/plot_design_matrix.py

```diff
@@ -114,6 +114,92 @@
 plt.show()
 
 
+# %%
+# Correlation between regressors
+# ------------------------------
+from matplotlib.colorbar import make_axes
+
+
+def plot_corr_matrix_mpl(
+    des_mat, partial="upper", cmap="bwr", ax=None, fig=None
+):
+
+    if not isinstance(des_mat, pd.DataFrame):
+        raise TypeError(
+            f"'des_mat' must be a andas dataframe instance.\n"
+            f"Got: {type(des_mat)}"
+        )
+
+    ALLOWED_CMAP = ["RdBu_r", "bwr", "seismic_r"]
+    if cmap not in ALLOWED_CMAP:
+        raise ValueError(f"cmap must be one of {ALLOWED_CMAP}")
```

Comment:
limiting to some divergent colormaps that have fairly good psychometric properties